Codecademy/gamut
# chore: nx testing integration

### Overview





### PR Checklist


#### Testing Instructions

1. Pull down the branch, run `yarn` and run the test suite `yarn test`
2. Run any other local commands you normally use to make sure they still work
3. look through the package diffs below for any breaking changes that were missed

Package Diffs:

https://diff.intrinsic.com/eslint-plugin-gamut/2.0.1/2.0.2-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/gamut-icons/7.5.0/7.5.1-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/gamut-illustrations/0.28.5/0.28.6-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/gamut-labs/33.0.7/33.0.8-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/gamut-patterns/0.8.11/0.8.12-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/gamut-styles/14.0.2/14.0.3-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/gamut-tests/4.0.10/4.0.11-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/gamut/50.1.0/50.1.1-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/macros/3.0.1/3.0.2-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/markdown-overrides/0.9.8/0.9.9-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/monaco/0.6.66/0.6.67-alpha.f58fb6.0
https://diff.intrinsic.com/@codecademy/variance/0.20.2/0.20.3-alpha.f58fb6.0

Expected changes:
1. extraneous files have been removed from the dist folders, including tests and configuration files
4. Minor non-functional code changes, mainly to things like d.ts files due to slight typescript config changes
5. Some missing source maps in the `dist` folders

Unexpected Changes:
1. Major code changes
3. Removed files from the `dist` directories (keep a look out for non-js files that are missing, I changed how we copy them into the `dist` folders.



[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/31922 'Named link title')  | [Monolith Env](https://github.com/codecademy-engineering/Codecademy/pull/31922 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/portal-app/pull/2351 'Portal App PR')  | [Portal Env](https://github.com/codecademy-engineering/portal-app/pull/2351 'Portal App ENV')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |
